### ISSUE TITLE
Any planned update or this tool is considered complete?

### ISSUE THREAD

**Issue by Pocokk** [SuiMachine/Silent-Hill-4---Tool#1]
Title. 

Just came across this program of yours and I was wondering if you plan to continue or simply revisit and perhaps make more adjustments, features or this is a complete version? 

Thank you!

**Comment by SuiMachine** [comment]
No, I am not planning on working on it, unless my upcoming playthrough catches something. I think there exist now better solutions written by ThirteenAG, Nemesis or Steam006 anyway.
